libs/components/src/lib/file-picker/file-picker.scss

```
@@ -0,0 +1,143 @@
+@use "partials/variables" as variables;
+
+@use "../../../../../dist/libs/tokens/scss/tokens.constants" as constants;
+
+@use "../../../../../dist/libs/tokens/scss/size.variables" as size;
+
+$low-ink-color: --_low-ink-color;
+
+.base {
+	display: inline-flex;
+	flex-direction: column;
+	inline-size: var(#{variables.$file-picker-inline-size}, 400px);
+}
+
+.control {
+	position: relative;
+	display: grid;
+	box-sizing: border-box;
+	flex-grow: 1;
+	align-items: center;
+	justify-content: center;
+	padding: 16px;
+	border: 1px dashed var(--vvd-color-neutral-400);
+	background-color: var(#{constants.$vvd-color-cta-50});
+	block-size: var(#{variables.$file-picker-block-size}, 52px);
+	border-radius: 6px;
+	color: var(#{constants.$vvd-color-neutral-600});
+	cursor: pointer;
+	font: var(#{constants.$vvd-typography-base});
+	inline-size: 100%;
+	outline: none;
+}
+.main {
+	pointer-events: none;
+}
+.icon {
+	color: var(#{constants.$vvd-color-neutral-600});
+	font-size: 24px;
+}
+
+.dz-preview {
+	$start: var(#{constants.$vvd-color-information-200});
+	$end: var(#{constants.$vvd-color-cta-600});
+
+	display: grid;
+	box-sizing: border-box;
+	padding: 10px;
+	border-radius: 6px;
+	grid-template-columns: 1fr auto;
+	grid-template-rows: auto auto;
+	inline-size: 100%;
+
+	.dz-details {
+		display: flex;
+		overflow: hidden;
+		flex-direction: column;
+		max-inline-size: 100%;
+
+		.dz-filename {
+			overflow: hidden;
+			color: var(#{constants.$vvd-color-canvas-text});
+			font: var(#{constants.$vvd-typography-base});
+			text-overflow: ellipsis;
+			white-space: nowrap;
+		}
+
+		.dz-size {
+			order: 1;
+			color: var(#{constants.$vvd-color-neutral-600});
+			font: var(#{constants.$vvd-typography-base-condensed});
+		}
+	}
+
+	&:not(.dz-success) {
+		.dz-size {
+			display: none;
+		}
+	}
+
+	&:not(.dz-error) {
+		.dz-progress {
+			display: block;
+			overflow: hidden;
+			background: var(#{constants.$vvd-color-neutral-100});
+			block-size: 6px;
+			border-radius: 3px;
+			inline-size: 100%;
+			margin-block-start: 8px;
+
+			.dz-upload {
+				display: block;
+				background: linear-gradient(to right, $start, $end);
+				block-size: 100%;
+				transition: width 300ms ease-in-out;
+			}
+		}
+	}
+
+	.dz-error-message {
+		color: var(#{constants.$vvd-color-alert-500});
+		font: var(#{constants.$vvd-typography-base-condensed});
+		margin-block-start: 4px;
+	}
+
+	&.dz-success {
+		.dz-progress,
+		.dz-error-message {
+			display: none;
+		}
+	}
+
+	&.dz-error {
+		border: 1px solid var(#{constants.$vvd-color-alert-500});
+
+		.dz-details .dz-size,
+		.dz-progress {
+			display: none;
+		}
+	}
+
+	.remove-btn {
+		display: inline;
+		align-self: center;
+		grid-column: 2 / -1;
+		grid-row: 1 / -1;
+	}
+}
+
+.message {
+	margin-block-start: 4px;
+	#{$low-ink-color}: var(#{constants.$vvd-color-neutral-600});
+}
+
+.preview-list {
+	&:empty {
+		display: none;
+	}
+
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+	margin-block-start: 4px;
```

```suggestion
	margin-block-start: 4px;
	overflow-y: auto;
```
if there are to many items :)